middleware-jackson/src/test/java/com/nytimes/android/external/store/middleware/jackson/JacksonReaderParserStoreTest.java

```
@@ -103,13 +103,13 @@ private void validateFoo(Foo foo) {
 
     @Test
     public void testNullJsonFactory() {
-        expectedException.expect(NullPointerException.class);
+        // Successful creation with a default factory is expected
```

These tests should fail. We want to fail early.

---

Ignore previous. Misread ☺️